src/containers/main-page/cv-form/components/fields/projects/components/CategorySelection.tsx

```diff
@@ -9,28 +9,71 @@ import AddCircleOutlineIcon from '@mui/icons-material/AddCircleOutline';
 
 import CircularSpinner from 'common/components/circular-spinner/circular-spinner';
 
+import { Category, Skill } from 'common/models/User';
 import { ProjectFieldValues } from '../utils/types';
 import { useUserFromDb } from '../../personal-information/lib/query-hooks';
 import { ButtonText, CategoryAddText, tooltipText } from './utils/constants';
 
-import { CategoriesTitleStyled, InfoIconStyled } from './utils/styled';
+import { ProjectTitleStyled, InfoIconStyled } from './utils/styledEdit';
 import SkillsToolsDialog from './SkillsToolsDialog';
+import CategoryItem from './CategoryItem';
+
+type CategoryItemProps = {
+  categories: {
+    category: string | undefined;
+    skill: string | undefined;
+    tools: string[] | undefined;
```

Comment:
Please remove `| undefined`  from everywgere here.
You may use Partial<T> utility type